recipe/recipe.yaml

```diff
@@ -12,15 +12,19 @@ source:
   patches: remove-getopt-required.patch
 
 build:
-  number: 1
+  number: 2
   script:
     # Get an updated config.sub and config.guess
-    - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+    - if: unix
+      then: cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+    - if: win
+      then: cmake %CMAKE_ARGS% -S . -B build
```

Comment:
Why isn't the Windows build getting installed at `PREFIX`?

Reply:
c.f. https://docs.conda.io/projects/conda-build/en/stable/resources/use-shared-libraries.html#shared-libraries-in-windows

For that weird `\Library` addendum, it seems to be some sort of packaging difference on windows. Anyway it seems  taken care of by the defaults correctly.

Reply:
TIL. Thanks @lgray!